app/components/Account/AccountVoting.jsx

```diff
@@ -757,6 +823,21 @@ class AccountVoting extends React.Component {
             </div>
         );
 
+        const hideLegacy = (
+            <div>
+                <input
```

Comment:
Please use a Switch from style-guide
`import {Switch} from "bitshares-ui-style-guide";`

For usage see e.g. https://github.com/blockchainprojects/bitshares-ui/blob/8f8cf007d4ed8aa7cb83aec018bf0b964cee502a/app/components/Exchange/Personalize.jsx#L220

If possible, put this switch in line with the proxy text field.

Reply:
fixed